doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst

```diff
@@ -43,7 +43,7 @@ Trusted Firmware-M (TF-M)
 
 Trusted Firmware-M provides a configurable set of software components to create a Trusted Execution Environment.
 It has replaced Secure Partition Manager as the solution used by |NCS| applications and samples.
-This means that when you build your application for ``_ns`` build targets, TF-M is automatically included in the build.
+This means that when you build your application for ``_ns`` board targets, TF-M is automatically included in the build.
```

Comment:
```suggestion
This means that when you build your application for ``*/ns`` board targets, TF-M is automatically included in the build.
```

Reply:
I think this was labeled a variant on the Board names page now. Should that terminology be used here as well?